### PR TITLE
perf(spec-parser): update list function signature

### PR DESCRIPTION
**Pull request by SLdragon**
- update list function signature
- show error message when user select multiple APIs with multiple auth/server url
- update export in index.browser.ts file

### CODE REVIEW

File: packages/fx-core/src/question/create.ts [hunk 1]
```diff
@@ -1559,22 +1560,61 @@ export function apiOperationQuestion(includeExistingAPIs = true): MultiSelectQue
...
           );
         }
+        const operations: ApiOperation[] = inputs?.supportedApisFromApiSpec as ApiOperation[];
+
+        const authNames: string[] = [];
+        const serverUrls: string[] = [];
+        let firstAuthName;
+        let firstServerUrl;
+        for (const inputItem of input) {
+          const operation = operations.find((op) => op.label === inputItem);
```
Comment (yuqizhou77) [inline packages/fx-core/src/question/create.ts:1578]: use "op.id". id is what returned from user inputs.

Comment (SLdragon) [reply packages/fx-core/src/question/create.ts:1578]: Thanks, will update

File: packages/fx-core/src/question/create.ts [hunk 2]
```diff
@@ -1559,22 +1560,61 @@ export function apiOperationQuestion(includeExistingAPIs = true): MultiSelectQue
...
+          if (operation) {
+            if (operation.authName) {
+              if (!firstAuthName) {
+                firstAuthName = operation.authName;
+                authNames.push(firstAuthName);
+              } else if (operation.authName !== firstAuthName) {
+                authNames.push(operation.authName);
```
[the remark this answers is not in the document]
Comment (SLdragon) [reply packages/fx-core/src/question/create.ts:1585]: Good idea, thanks~

**Review by yuqizhou77 — APPROVED** [review]

**Review by yuqizhou77 — DISMISSED** [review]

**Review by nliu-ms — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -297,6 +297,8 @@
   "core.createProjectQuestion.apiSpec.operation.title": "Select Operation(s) Teams Can Interact with",
   "core.createProjectQuestion.apiSpec.operation.placeholder": "GET/POST methods with at most one required parameter and no auth are listed",
   "core.createProjectQuestion.apiSpec.operation.invalidMessage": "%s API(s) selected. You can select at least one and at most %s APIs.",
+  "core.createProjectQuestion.apiSpec.operation.multipleAuth": "Your selected APIs have multiple authorizations %s which is not supported.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:300]: is -> are

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -297,6 +297,8 @@
...
+  "core.createProjectQuestion.apiSpec.operation.multipleServer": "Your selected APIs have multiple server URLs %s which is not supported.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:301]: is -> are

File: packages/fx-core/resource/package.nls.json (hunk 3 above)
Comment (SLdragon) [reply packages/fx-core/resource/package.nls.json:300]: Thanks, will update

**Review by yuqizhou77 — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**